tiledb/ml/readers/_tensor_schema.py

```diff
@@ -220,19 +193,22 @@ class SparseTensorSchema(TensorSchema):
 
     def __init__(
         self,
-        array: tiledb.Array,
-        key_dim: Union[int, str] = 0,
-        fields: Sequence[str] = (),
+        array_params: ArrayParams,
         transform: Optional[Callable[[Tensor], Tensor]] = None,
     ):
-        super().__init__(array, key_dim, fields, transform)
+        super().__init__(array_params)
         key_counter: Counter[Any] = Counter()
         key_dim = self._all_dims[0]
         query = self._array.query(dims=(key_dim,), attrs=(), return_incomplete=True)
         for result in query.multi_index[:]:
             key_counter.update(result[key_dim])
         self._key_range = InclusiveRange.factory(key_counter)
         self._query_kwargs["dims"] = self._all_dims
+        self.transform = transform
+
+    @property
+    def key_range(self) -> InclusiveRange[Any, int]:
```

Comment:
Duplicated property

Reply:
Why is this duplicated? It justs implements the `abstract` `key_range` of `Tensorschema` and it was already there it is not affected by this PR

Reply:
There are two identical `SparseTensorSchema.key_range`, probably a bad merge conflict resolution